packages/core/src/app/customer/CheckoutButtonContainer.tsx

```diff
@@ -63,11 +67,11 @@ const CheckoutButtonContainer: FunctionComponent<CheckoutButtonContainerProps &
         return null;
     }
 
-    if (isPaypalCommerce && isPaymentStepActive) {
-        return null;
-    }
-
     const renderButtons = () => availableMethodIds.map((methodId) => {
+        if (isPaymentStepActive && paypalCommerceIds.includes(methodId)) {
```

Comment:
Could you please convert `paypalCommerceIds.includes(methodId)` into  `isPaypalCommerce(methodId)` for readability and place it next to `sortMethodIds()`?

Reply:
Sure, I will update the PR with second commit message